lib/internal/event_target.js

```diff
@@ -50,7 +50,7 @@ class Event {
     this.#cancelable = !!cancelable;
     this.#bubbles = !!bubbles;
     this.#composed = !!composed;
-    this.#type = String(type);
+    this.#type = '' + type;
```

Comment:
```suggestion
    this.#type = `${type}`;
```

We have other places in core that do it like this

Reply:
Sounds nice to me, we should probably enforce this in the lint.